options.html

```diff
@@ -0,0 +1,410 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Monaco Markdown Editor For GitHub</title>
+  </head>
+  <body>
+    <p><label><input type="checkbox" id="scrollBeyondLastLine" data-setting="true"> Scroll beyond last line</label></p>
+    <p>
+      <label>Word wrap <select id="wordWrap" data-setting="true">
+          <option value="off">Off</option>
+          <option value="on">On</option>
+          <option value="wordWrapColumn">Word wrap column</option>
+          <option value="bounded">Bounded</option>
+        </select></label>
+    </p>
+    <p>
```

Comment:
These are a lot of settings. I think it might make sense to only add requested settings to keep the UI lean, what do you think?

Reply:
I assumed that if Microsoft put effort into implementing these settings, there are people that want them.

Reply:
I doubt vscode exposes all the those settings. Imo these settings are for programmers to customize Monaco to fit into the product, not necessarily for end users.
I'd love to hear more opinions on that.
My philosophy is usually "less is more", especially if I have to maintain it!

Maybe you can, alternatively, somehow reduce the redundancy in options.html? Ideally each setting should only correspond to a single line of code.

Reply:
> I doubt vscode exposes all the those settings. Imo these settings are for programmers to customize Monaco to fit into the product, not necessarily for end users.

Not for all settings, but when I was in doubt, I checked if VSCode provides the setting. So you're both right and wrong at the same time 😄.

> Maybe you can, alternatively, somehow reduce the redundancy in options.html? Ideally each setting should only correspond to a single line of code.

I had to type all of this stuff, so be sure I've thought about that 😆. I came up with the conclusion that there is nothing I can do to really reduce the number of HTML code here. By `data-setting=true` and automatically applying the setting based on the element ID, most of the work is done. I _can_ reduce some of the code by custom elements, but I don't think this is worth (The most of the code is from the dropdowns, where I have to specify the options either way).

Reply:
Experience was good and it reduced the number of lines. Pushed.